README.md

```diff
@@ -46,6 +46,8 @@ Usage
 8. Wait for Jenkins to restart.
 
 ## Generating a proper IAM user:
+Note : The following steps need to be followed if you are using Secret and Access key to access Device Farm.
```

Comment:
Follow these steps if you are using ......

Reply:
Done